assets/tests/components/detourListPage.test.tsx

```diff
@@ -2,11 +2,52 @@ import { describe, test, expect, jest } from "@jest/globals"
 import React from "react"
 import renderer from "react-test-renderer"
 import { DetourListPage } from "../../src/components/detourListPage"
+import { useAllDetours } from "../../src/hooks/useAllDetours"
 
-jest.useFakeTimers().setSystemTime(new Date("2024-08-16T13:00:00"))
+jest.useFakeTimers().setSystemTime(new Date("2024-08-29T20:00:00"))
+
+jest.mock("../../src/hooks/useAllDetours", () => ({
+  useAllDetours: jest.fn(() => {}),
+}))
```

Comment:
https://github.com/mbta/skate/pull/2757#discussion_r1738629215

Reply:
Ok so I had no trouble applying this comment to the `addDetour` test suite, but did run into issues here, and I can't gather why. The below change leads to an empty snapshot.

```
beforeEach(() => {
  jest.mocked(fetchDetours).mockReturnValue(neverPromise())
})

describe("DetourListPage", () => {
  test("renders detour list page", () => {
    jest.mocked(fetchDetours).mockResolvedValue(Ok({ ...
```

Reply:
I _think_ part of the issue _may_ be using `renderer.create` and the other is that now that we're mocking a _promise_, we have to wait for the table to show up, so you'll need `await screen.findBy<something>()` before the snapshot

Reply:
Marvelous, the combined changes covered it. Thank you 🙏